src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/TestEndpoints.cs

```diff
@@ -212,5 +260,9 @@ internal TestEndpoints(TestEndpoints testEndpoint, ConnectionString connString):
         // Graph Token Audience 
         public Uri GraphTokenAudienceUri { get; set; }
 
+        public Uri PublishingSettingsFileUri { get; set; }
+
+        //public Uri AADServiceUri { get; set; }
```

Comment:
yes, and this would be the uri for aad authority, without the tenant id.  Where is this value being mapped if not here?